java/system-test/system-test-drivers/src/test/java/sleeper/systemtest/drivers/testutil/LocalStackTestInstance.java

```diff
@@ -58,6 +59,7 @@ private static DeployInstanceConfiguration buildConfigurationWithTableProperties
         properties.set(FORCE_RELOAD_PROPERTIES, "true");
         properties.set(DEFAULT_DYNAMO_STRONGLY_CONSISTENT_READS, "true");
         properties.set(RETAIN_INFRA_AFTER_DESTROY, "false");
+        properties.set(RETAIN_LOGS_AFTER_DESTROY, "false");
```

Comment:
To clarify, this also needs changing to address my previous comment: https://github.com/gchq/sleeper/pull/5965#discussion_r2494822761

> We need to keep the logs after a system test instance is destroyed, otherwise we wouldn't be able to see what happened in the nightly tests.